Sources/PackageModel/UserToolchain.swift

```diff
@@ -190,7 +188,18 @@ public final class UserToolchain: Toolchain {
         if let librarian = try? UserToolchain.getTool(tool, binDirectories: binDirectories) {
             return librarian
         }
-        return try UserToolchain.findTool(tool, envSearchPaths: searchPaths, useXcrun: useXcrun)
+        if triple.isApple() || triple.isWindows {
+            return try UserToolchain.findTool(tool, envSearchPaths: searchPaths, useXcrun: useXcrun)
+        } else {
+            if let librarian = try? UserToolchain.findTool(tool, envSearchPaths: searchPaths, useXcrun: false) {
+                return librarian
+            }
+            // Fall back to looking for binutils `ar` if `llvm-ar` can't be found.
```

Comment:
I don't think this fallback is necessary now that `llvm-ar` ships with all Swift toolchains, but I just followed the TODO comment I removed exactly for now.

Reply:
Which versions are we talking about? In theory, one can mix-and-match compilers and SwiftPM, so it would be unfortunate if we're cutting off versions that are recent.

Reply:
If we can support 5.8 and 5.9, that seems good enough, thanks.

Reply:
Removed the fallback as you suggested.